src/kimchima/tests/test_pipelines_factory.py

```diff
@@ -60,3 +60,38 @@ def test_text_generation(self):
     
             self.assertIsNotNone(pipe)
             self.assertEqual(pipe.task, 'text-generation')
+
+        @classmethod
+        def test_customized_pipe(self):
+            """
+            Test customized_pipe method
+            """
+    
+            pipe = PipelinesFactory.customized_pipe(
+                task="text-generation",
+                model=self.model,
+                tokenizer=self.tokenizer,
+                streamer=self.streamer,
+                max_new_tokens=20,
+                device_map='auto'
+                )
+    
+            self.assertIsNotNone(pipe)
+            self.assertEqual(pipe.task, 'text-generation')
+
+        @classmethod
+        def test_chat_response(self):
+            """
+            Test chat_response method
+            """
+            conversation_model="facebook/blenderbot-400M-distill"
```

Comment:
We are trying to download `facebook/blenderbot-400M-distill` almost 750 MB in test case is not a good idea. I think we can keep use `gpt2` with same pipeline obj